aiida/cmdline/params/options/interactive.py

```diff
@@ -191,10 +191,12 @@ def safely_convert(self, value, param, ctx):
         successful = False
         try:
             value = self.type.convert(value, param, ctx)
+            value = self.after_callback(ctx, param, value)
```

Comment:
as mentioned in my previous comment, looking at the definition of `def prompt_callback`  this seems to result in `after_callback` being called twice in the last run.
Is this correct?

Reply:
I think you were right. I added it because if an invalid value was passed twice in a row in interactive mode, the second time, the callback was not called and the value was accepted. However, the correct approach is to make sure the normal `callback` is called, which will ensure that the prompt loop is reentered properly. Now I think the `after_callback` is only called once.